docs/content/theory.rst

```diff
@@ -14,11 +13,119 @@ Here's how it works.
     :height: 500px
     :align: center
 
-Matrix modifications
-=======
+User input 
+~~~~~~~~~~~~~~
+
+``timex_lca`` requires 3 inputs:
+ 
+1. a static foreground system model with
+2. temporal information using the attribute ``temporal_distribution`` on technosphere or biosphere exchanges in the foreground system modeel, and 
+3. a set of background databases, which must have a reference in time.`
+
+.. raw:: html
+
+    <details>
+        <summary>
+            <span style="color: white; padding: 4px; cursor: pointer;">ℹ️</span>
+            <strong style="margin-left: 8px;">More info on inputs</strong>
+        </summary>
+        <div style="background-color: #f0f7ff; padding: 10px;">
+            <ul>
+                <li>The foreground system must have exchanges linked to one of the background databases. These exchanges at the intersection between foreground and background databases will be relinked by <code>timex_lca</code>.</li>
+                <li>Temporal distributions can occur at technosphere and biosphere exchanges and can be given in various forms, see <a href="https://github.com/brightway-lca/bw_temporalis/tree/main">BW Temporalis documentation</a>, including absolute (e.g. 2024-03-18) or relative (e.g. 3 years before) nature and can have different temporal resolution (down to seconds but later aggregation supports resolutions down to hours).</li>
+                <li>Temporal distributions are optional. If none are provided, no delay between producing and consuming process is assumed and the timing of the consuming process is adopted also for the producing process.</li>
+            </ul>
+        </div>
+    </details>
+
+    <script>
+        document.addEventListener("DOMContentLoaded", function() {
+            var details = document.querySelectorAll("details");
+            details.forEach(function(detail) {
+                detail.querySelector("summary").addEventListener("click", function() {
+                    this.parentElement.classList.toggle("open");
+                });
+            });
+        });
+    </script>
+
+Graph traversal
+~~~~~~~~~~~~~~~~
+.. _`BW_temporalis`: https://github.com/brightway-lca/bw_temporalis/tree/main
+``timex_lca`` uses the graph traversal from `BW_temporalis`_ to propagate the temporal information along the supply chain. The graph traversal is priority-first, following the most impactful node in the graph based on the static pre-calculated LCIA score for a chosen impact category. 
+All input arguments for the graph traversal, such as maximum calculation count or cut-off, can be passed to the ``TimexLCA`` instance.
+
+By default, only the foreground system is traversed, but nodes to be skipped during traversal can be added to a ``edge_filter_function``. 
```

Comment:
not really "added" but rather "specified by a filter function"

Reply:
sure!